.github/workflows/build-linux.yml

```diff
@@ -1,11 +1,12 @@
 
-name: Build on Ubuntu
+name: Build GlassFish on Ubuntu
 on:
   pull_request:
     branches: [ "master", "7.0", "8.0" ]
 jobs:
   build:
     runs-on: ubuntu-24.04
+    name: Build on Ubuntu
```

Comment:
In all other places you use `Build` **GlassFish** `on {system_name}`. Maybe here too?

Reply:
Sure, I missed it here. Thanks for pointing this out.